### PR TITLE
Random visual tweaks to MFA components

### PR DESCRIPTION
**Pull request by keithpitt**
A bunch of random stuff in here! I tightened some stuff up and made the recovery code boxes consistent between viewing and saving them.

A few notable changes:

- [x] Use the rad icon in the dialogs (because it looks rad)
- [x] Show "Step 1", "Step 2"
- [x] Added a "Download" button for recovery codes
- [x] Moved the "you're reconfiguring" message to the top of the dialog

### Before

image
image
image
image

### After

image
image
image
image

### CODE REVIEW

**Comment by keithpitt** [comment]
image

I also sharpened up the secret code thingy here. I figured that the dialog was just devoting a lot of space to a (sometimes thing). The barcode and input thingy at the bottom are too nice, so I wanted to reduce the impact the of the secret key on them.

**Comment by keithpitt** [comment]
Oh! And after googling, peeps call them "barcode" so I just updated the name (instead of QR code)

File: app/components/user/TwoFactor/TwoFactorConfigure/TwoFactorConfigureActivate/index.js [hunk 1]
```diff
@@ -63,40 +60,17 @@ export default class TwoFactorConfigureActivate extends React.PureComponent<Prop
...
-              <strong>Secret Key</strong>
-            </div>
-            <div className="dark-gray mb1">
-              Can’t use the QR code? You can use this secret key instead.
-            </div>
-            <div className="flex">
-              <div className="input mr2 monospace" style={{ lineHeight: 1.8 }}>
-                {provisioningUriSecret}
+              <div className="dark-gray center mt2">
+                Can’t use the barcode?<br />Type in this secret key instead: <span className="monospace rounded" style={{ fontSize: 13 }}>{provisioningUriSecret}</span>
```
Comment (eleanorakh) [inline app/components/user/TwoFactor/TwoFactorConfigure/TwoFactorConfigureActivate/index.js]: ah, I thought this had to be the whole URI and not just the secret?

Comment (keithpitt) [reply app/components/user/TwoFactor/TwoFactorConfigure/TwoFactorConfigureActivate/index.js]: So did I originally, turns out I was telling lies!

**Pull request merged**